index.js

```diff
@@ -148,6 +226,120 @@ types[MAESTRO] = {
   }
 };
 
+types[ELO] = {
+  niceType: 'Elo',
+  type: ELO,
+  prefixPattern: new RegExp('^(' +
+    '[4-6]' +
+
+    '|' +
+
+    '4[035]|4[035]1' +
+    '|' +
+    '4011|40117|40117[89]' +
+    '|' +
+    '4312|43127|431274' +
+    '|' +
+    '438|4389|43893|438935' +
+    '|' +
+    '4514|45141|451416' +
+    '|' +
+    '457|457[36]|45739|45763|457393|45763[12]' +
+
+    '|' +
+
+    '50|50[69]' +
+    '|' +
+    '506[6-7]|50669|5067[0-7]|5067[0-6][0-9]|50677[0-8]' +
```

Comment:
I know `[0-9]` is equivalent to `\d`, but I felt like having the range made this (slightly) easier to read.